packages/apollo-server-core/src/errors.ts

```diff
@@ -133,3 +140,40 @@ export class ForbiddenError extends ApolloError {
     super(message, 'FORBIDDEN');
   }
 }
+
+export function formatApolloErrors(
+  errors: Array<Error>,
+  options?: {
+    formatter?: Function;
+    logFunction?: LogFunction;
+    debug?: boolean;
+  },
+): Array<Error> {
+  const { formatter, debug, logFunction } = options;
+  return errors.map(error => enrichError(error, debug)).map(error => {
+    if (formatter !== undefined) {
```

Comment:
Rather than having this larger `if` block surrounding the totality of this `map`, could we do something like:

```js
  if (!formatter) {
    return error;
  }

  // Remainder of existing logic, indented by two less spaces.
  try {
     // ...
```
?

By hoisting that logic, it allows the reader of this code to focus on the remaining functionality, rather than needing to keep in mind that we're still inside a conditional block (the `if`).